assets/src/edit-story/components/panels/design/animation/stories/directionRadioInput.js

```diff
@@ -30,7 +30,7 @@ import {
 import { DirectionRadioInput } from '../directionRadioInput';
 
 export default {
-  title: 'Animations/Direction Radio Input',
+  title: 'Stories Editor/Components/Panels/Animations/Direction Radio Input',
```

Comment:
just thought these make more sense now in `Stories Editor/Components` not `Animations`

Reply:
yea completely agree. Actually mirrors the file structure in the app. I think if you're looking at the story book implementation now it actually gives you some insight on how to find it in the app. good call